runtime/compiler/optimizer/IdiomTransformations.cpp

```diff
@@ -740,8 +741,8 @@ IANDSpecialNodeTransformer(TR_CISCTransformer *trans)
       }
    return ret;
    }
-
 //////////////////////////////////////////////////////////////////////////
+
```

Comment:
It looks like the location of the blank line moved.  May I ask you to move it above the line with all slashes?

```
   }

//////////////////////////////////////////////////////////////////////////
// utility routines

static void
```